Tribler/Test/Core/TFTP/test_tftp_handler.py

```diff
@@ -122,7 +122,8 @@ def test_function():
             test_function.is_called = True
             return False
         test_function.is_called = False
-        self.handler.session.get_torrent_store = test_function
+        self.handler.session.config = MockObject()
+        self.handler.session.config.get_torrent_store_enabled = test_function
```

Comment:
Thanks for the feedback, but this function name was introduced in https://github.com/Tribler/tribler/pull/2944. I can make a PR to change that if you like.